src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml

```diff
@@ -0,0 +1,101 @@
+<documentation title="Array Indent">
+    <standard>
+    <![CDATA[
+    The opening square bracket/parenthesis of a multi-line array must be indented at least to the same level of the start of the statement.
```

Comment:
I opted to use `opening square bracket/parenthesis` here and in a few other places as the sniff supports both array syntaxes. That is not super correct as in the old syntax, the opening token of an array is not just a parenthesis, but actually `array(`.

Reply:
What about using the phrase "open brace" (and "close brace" for the end, of course) ? Would that work ? What do you think ?